angular/src/app/landing/data-files/data-files.component.ts

```diff
@@ -237,24 +246,19 @@ export class DataFilesComponent implements OnInit, OnChanges {
 
         let count = 0;
         let downloadedCount = 0;
-        let inCartCount = 0;
         let root: TreeNode = { data: { name: '', key: '' }, children: [] };
         let node: TreeNode = null;
-        let cartitem: DataCartItem = null;
+
+        // Filter out sha files
         for (let comp of this.record['components']) {
             if (comp.filepath && comp['@type'].filter(tp => tp.includes(':Hidden')).length == 0) {
-                node = insertComp(comp, root);
-                /*
-                cartitem = this.globalDataCart.findFile(this.record['@id'], comp.filepath);
-                if (cartitem) {
-                    node.data.isInCart = true;
-                    inCartCount++;
-                    node.data.downloadStatus = cartitem.downloadStatus;
-                }
-                */
-                if (node.data.comp['@type'].filter(tp => tp.endsWith("File")).length > 0) {
-                    count++;
-                    if (node.data.downloadStatus == DownloadStatus.DOWNLOADED) downloadedCount++;
+                if(!(comp['downloadURL'] && comp['downloadURL'].endsWith('sha256'))){
```

Comment:
I would prefer the test be based on the file type (i.e. excluding ChecksumFile types) rather than on the file extension.  There can be checksum files that contain a different type (e.g. "MD5" or "SHA512"), and there can be checksum files provided explicitly by the author and should be considered the data file.  The publishing process determines which are which and sets the `@type` property accordingly.  

I can make and check-in this fix.